recipe/meta.yaml

```diff
@@ -21,12 +21,13 @@ requirements:
   build:
     - pkg-config  # [not win]
     - python
-    - zeromq      # [not win or (win and py35)]
+    - zeromq      # [not win]
     - libsodium   # [not win]
   run:
     - python
-    - zeromq      # [not win or (win and py35)]
+    - zeromq      # [not win]
     - libsodium   # [not win]
+    - vs2015_runtime  # [win and py35]
```

Comment:
I thought this was the cause of the recent failures, but it doesn't appear to be. Removed, and we'll see how it goes!

What would happen if a user got Python 3.5 from `defaults`? Would they still get vs2015_runtime?